src/lib/gitutils.ts

```diff
@@ -109,14 +109,19 @@ export const pushBranch = async (branchName: string): Promise<void> => {
 
 /**
  * Gets the origin url.
+ * @param absRepositoryPath The Absolute Path to the Repository to fetch the origin
  */
-export const getOriginUrl = async (): Promise<string> => {
+export const getOriginUrl = async (
+  absRepositoryPath?: string
```

Comment:
can change to `absRepositoryPath = "."` to give it the default value since thats what you already do below